go.mod

```diff
@@ -17,4 +17,5 @@ require (
 	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd // indirect
 	golang.org/x/sys v0.0.0-20220712014510-0a85c31ab51e
 	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Is it possible to remove dependency on the v2 package altogether?

Reply:
yes, but that build failed and the method I choose for problem solving was splitting the changes